rust/rsc/src/bin/rsc/config.rs

```diff
@@ -15,8 +15,20 @@ pub struct RSCTTLConfig {
     pub tick_rate: u64,
     // How long an object is allowed to live
     pub ttl: u64,
-    // Maximum number of objects to delete at a time. Must be 1 >= x <= 16000
+    // Maximum number of objects to delete from the db at a time. Must be 1 >= x <= 16000
+    pub chunk_size: u32,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct RSCBlobTTLConfig {
+    // How often to run the eviction check in seconds
+    pub tick_rate: u64,
+    // How long an object is allowed to live
+    pub ttl: u64,
+    // Maximum number of objects to delete from the db at a time. Must be 1 >= x <= 16000
```

Comment:
```suggestion
    // Maximum number of blobs to delete from the db at a time. Must be 1 >= x <= 64000
```

Reply:
Not sure what the actual limit is but we've tested at least 64k so might as well note that.